lms/djangoapps/instructor/tests/test_spoc_gradebook.py

```diff
@@ -90,6 +86,10 @@ class TestDefaultGradingPolicy(TestGradebook):
     Tests that the grading policy is properly applied for all users in the course
     Uses the default policy (50% passing rate)
     """
+    def test_response_code(self):
```

Comment:
I think it would be better to put this assert into the setUp function itself, since it is checking that the setUp succeeded.